### PR TITLE
Add supervisor

### PR DESCRIPTION
**Pull request by kalefranz**

### CODE REVIEW

**Comment by jakirkham** [comment]
Looks like `uwsgi` slipped into this PR. Does it need to be here? If not, can we please remove it as it has its own PR ( https://github.com/conda-forge/staged-recipes/pull/478 )?

**Comment by kalefranz** [comment]
Yeah uwsgi sneaking in was definitely a mistake. Fixed.

**Comment by jakirkham** [comment]
No worries. Thanks for fixing that.

**Comment by kalefranz** [comment]
Ok this recipe is ready to go from my perspective.

File: recipes/supervisor/post-link.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+if [[ $(id -u) == 0 ]]; then
+    mkdir -p /etc/supervisord/ && ln -s $PREFIX/etc/supervisord/ /etc/supervisord/
+    ln -s $PREFIX/etc/supervisord.conf /etc/supervisord.conf
```
Comment (msarahan) [inline recipes/supervisor/post-link.sh]: This is going to require sudo, right?  How can we make that gel with conda's user-level acess model?  Generally, if we're telling people to use sudo, they'll end up with files owned by root, which breaks later usability, then they devolve into using sudo for everything.  Imagine

    sudo ipython

Is there a way to not require sudo here?  I understand that it probably means not using supervisor at the system level - but it is still useful at the user level.  I use it for my raspberry pi photo frame at home this way.

Comment (kalefranz) [inline recipes/supervisor/post-link.sh]: No, actually it's not.  The post-install shouldn't be doing this linking if you're not running as root.  So if you've installed miniconda as root, then it'll link for you.  If not, then you just get your file structure in your $PREFIX.

Comment (kalefranz) [inline recipes/supervisor/post-link.sh]: This is kind of uncharted territory actually.  It's still a question for me what the "best practice" should be for conda doing OS management tasks like directory creation (outside of $PREFIX), user creation, setting permissions, etc.  The native OS package managers do all of this when they install packages; conda should too.  At some point.

This part is just a little nudge in that direction.  If you've already installed conda as root, then go ahead and link these files into the `/` file structure.

Comment (msarahan) [inline recipes/supervisor/post-link.sh]: I see.  Yes, I think your scripts here are a good way to do what you want here, and making it work both on the user-level and root level is a good idea.

+1 to merge from me.

**Comment by msarahan** [comment]
Merging.  Thanks @kalefranz

**Pull request merged**